repository/p.json

```
@@ -1589,6 +1589,17 @@
 				}
 			]
 		},
+		{
+			"name": "Piranha",
+			"details": "https://github.com/Kostasn2/piranha-syntax-highlighting-package",
+			"labels": ["piranha", "syntax highlight"],
```

```suggestion
			"labels": ["piranha", "language syntax"],
```

"language syntax" is the canonical label for a package like this.

---

Yea sure, you are right! Just changed it!